### PR TITLE
Create Get-WhiteboardOwners

### PR DESCRIPTION
**Pull request by shlevari**
Added documentation for the Get-WhiteboardOwners powershell module.

### CODE REVIEW

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardOwners.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## SYNOPSIS
+
+Gets all the users in a tenant who own whiteboards in a specified geography.
+
+## SYNTAX
+
+### Get the whiteboard owners in a geography
+
+```powershell
+Get-WhiteboardOwners -Geography <geography> -ContinuationToken <optionalToken> [<CommonParameters>]
```
Comment (get-itips) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardOwners.md:24]:
```suggestion
Get-WhiteboardOwners [-Geography] <String> [[-ContinuationToken] <String>] [-ForceAuthPrompt]
 [<CommonParameters>]
```

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardOwners.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ContinuationToken:         continuation token
+```
+
+
+## PARAMETERS
+
+### -Geography
+Required. The geography to look for board owners in. Accepted values are: Europe, Australia, or Worldwide (all boards not in australia or europe).
+
+```yaml
```
Comment (get-itips) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardOwners.md:56]: Please see this, it is the output given by PlatyPS for the Geography parameter

image

**Review by get-itips — CHANGES_REQUESTED** [review]
Can you run PlatyPS against the cmdlet and correct the parameters yaml? Thank you

**Comment by yogkumgit** [comment]
@shlevari review and implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@shlevari Gentle reminder

**Pull request merged**